docs/source/deployguide/ca-config.md

```diff
@@ -0,0 +1,417 @@
+
+If this is not a desired configuration for your CA, you can leave the value of this parameter blank.
+
+## intermediate CA
+```
+intermediate:
+  parentserver:
+    url:
+    caname:
+
+  enrollment:
+    hosts:
+    profile:
+    label:
+
+  tls:
+    certfiles:
+    client:
+      certfile:
+      keyfile:
+```
+
+Intermediate CAs are not required, but to reduce the risk of your root CA becoming compromised, you may want to include one or more intermediate CAs in your network.
+
+**Important:** Before setting up an intermediate CA, you need to verify that the `csr.ca.pathlength` parameter in the parent CA, is at least `1`. When set to `1`, the root CA can issue intermediate CA certificates, but these intermediate CAs may not in turn enroll other intermediate CAs. If you want your intermediate CA to be able to enroll other intermediate CAs, the root ca `pathlength` needs to be set to `1`.
+
+- **`parentserver.url`**: Specify the parent server url in the format `https://<PARENT-CA-ENROLL-ID>:<PARENT-CA-SECRET>@<PARENT-CA-URL>:<PARENT-CA-PORT>`.
+- **`parentserver.caname`**: Enter `ca.name` of the parent CA server.
+- **`enrollment.profile:`**: Enter the value of the `signing.profile` for the parent CA. Normally this would be `ca` unless this is an intermediate TLS CA server.
+- **`tls.certfiles`**: Enter the location and name of the parent CA ca-cert.pem file. For example, `tls/ca-cert.pem`. This location is relative to where the configuration .yaml file exists.
+
+In addition to editing this `intermediate` section, you also need to edit the following sections of the configuration .yaml file for this intermediate CA:
+- `csr` - Ensure that the `csr.cn` field is blank.
+- `port` - Be sure to set a unique port for the intermediate CA.
+
+## port
+```
+port:
+```
+
+Each CA must be running on its own unique port and obviously must not conflict with any other service running on that port. You need to decide what ports you want to use for your CAs ahead of time and configure that port in the .yaml file.
+
+## user registry
+```
+registry:
+  # Maximum number of times a password/secret can be reused for enrollment
+  # (default: -1, which means there is no limit)
+  maxenrollments: -1
+
+  # Contains identity information which is used when LDAP is disabled
+  identities:
+     - name: <<<adminUserName>>>
+       pass: <<<adminPassword>>>
+       type: client
+       affiliation: ""
+       attrs:
+          hf.Registrar.Roles: "*"
+          hf.Registrar.DelegateRoles: "*"
+          hf.Revoker: true
+          hf.IntermediateCA: true
+          hf.GenCRL: true
+          hf.Registrar.Attributes: "*"
+          hf.AffiliationMgr: true
+```
+Before a certificate can be generated for an identity, the associated user must be registered. If you are not using an LDAP user registry, then this section along with the associated registry database `db:` section are required.
```

Comment:
I would introduce LDAP and maybe link to the section the first time it is mentioned.

Reply:
Good suggestion. Done.